### ISSUE TITLE
STK 1.3 monitor goes black, input unresponsive, can hear sound

### ISSUE THREAD

**Issue by kalbfled** [supertuxkart/stk-code#4634]
## Description

I have STK installed on Ubuntu 20.04 as a snap package, and it automatically updated to version 1.3 yesterday.  Twice already while playing the game, seemingly at a random time, my monitor has gone black and given me the "Checking for signal..." dialog.  No input has any effect, but I can continue to hear the game music.  I have to press and hold the power button to turn of my computer, and I get prompted to reset the BIOS upon rebooting.

## Steps to reproduce

I don't know how other people can reproduce this.  It might be idiosyncratic to my machine or not related to STK at all, but I doubt it.  There have not been any other changes to my system.  I'm opening the issue in case other people are seeing the same behavior.

## Configuration

STK release version: 1.3
STK Source (PPA, distribution package, official binary, etc.):  Ubuntu 20.04 software center snap install
System:  Ubuntu 20.04
Graphics card: Nvidia GTX 980
CPU: Intel Core i7-5820K CPU @ 3.30GHz × 12 
Gamepads/keyboards models if related to the issue: not related

## Additional Information

I attached what I think are the relevant system log entries.
[system_log_gpu_messages.txt](https://github.com/supertuxkart/stk-code/files/7269467/system_log_gpu_messages.txt)

**Comment by mrkubax10** [comment]
Does this also happen using official package (not from snap)?

**Comment by kalbfled** [comment]
By "official package," do you mean the non-snap version in the Ubuntu 20.04 software center (which is version 1.1), or do you mean 1.3 as downloadable from the STK website?

This has never happened on my computer before, including while playing the snap package of STK 1.1 and 1.2.  I don't think it's a temperature issue either.  A couple of weeks ago, my computer ran at 100% GPU and CPUs for 3.5 days to render an animation with Blender, and that did not have any adverse effects.

**Comment by mrkubax10** [comment]
> By "official package," do you mean the non-snap version in the Ubuntu 20.04 software center (which is version 1.1), or do you mean 1.3 as downloadable from the STK website?

Downloadable from STK website

**Comment by kalbfled** [comment]
I downloaded v1.3 from the website and had the same issue.  I entered story mode, completed a challenge on the default graphics settings, set the graphics to 6 (highest settings; same as I've played previous versions), and completed another challenge.  At the start of the 3rd challenge, the screen went black.

I'm 99% sure this is an STK issue, although I can't say if it's an STK bug or some kind of incompatibility with my system after a routine update.  I used my computer all day yesterday without incident, but I'm 3 for 3 on blackouts when I play STK v1.3.

I suppose I could download v1.2 and see if that still works, but that seems unhelpful because I know it worked 2 or 3 days ago.

Curiously, I didn't have anything in the logs about "GPU fell off the bus" after this most recent crash.  I'm not sure if that's a consequence of running the game from the command line rather than clicking on the launch icon installed with the snap.
